.kokoro/samples/python3.6/presubmit.cfg

```diff
@@ -0,0 +1,33 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Not a big deal, but it seems like this file has some redundant configs.

Reply:
Updated